war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -1958,7 +1958,9 @@ function replaceDescription(initialDescription, submissionUrl) {
   }
   fetch("./descriptionForm", {
     method: "post",
-    headers: crumb.wrap({}),
+    headers: crumb.wrap({
+      "Content-Type": "application/x-www-form-urlencoded",
+    }),
```

Comment:
Here the body is using our own private equivalent to `URLSearchParams`, so it is necessary to set the header.